packages/Liquid-Core.package/LiquidPoll.class/instance/initialize.st

```diff
@@ -2,4 +2,5 @@ initialize
 initialize
 	answerSets := OrderedCollection new.
 	self id: UUID new.
-	isOpen := true.
\ No newline at end of file
+	isOpen := true.
+	guard := LQAcceptingGuard new.
```

Comment:
[sand?] What exactly is the reason we have a `LQAcceptingGuard`? Do you have an example of where we use/need it in the code?

Reply:
`LQAcceptingGuard` implements the Null-Object pattern. As long as an object doesn't have a password set (e.g. during testing, where we don't want to concern ourselves with stupid passwords), the `LQAcceptingGuard` accepts *any* password it's given. Thus, it's an *accepting* guard.